### ISSUE TITLE
Key Vault Admin and Readers

### ISSUE THREAD

**Issue by JFolberth** [Azure/Databricks360#24]
`2.1.3 edit the config yaml files found in /iac-adb-360/pipelines/azure/configdev.yml and configprd.yml to reflect the correct Resource Group name, location and the group object id's for the kvadmins as well as the kvsecretreaders. This groups are added as keyvault admins and keyvault secret readers in the key vault, which is going to be automatically deployed by the pipeline. So if not already done so, you'll have to create two groups p.ex. kvadmins and kvsecretreaders in Microsoft Entra ID, retrieve the OIDs and edit them in the two files accordingly. After you edit this files, they need to be updated in the repo. So if you were working locally, you'll have to commit and push to the repo.(make sure you are on the 'dev' branch)
`
I am not sure the purpose of this? Is it to assign the service principals the roles? If so wouldn't that be done directly against the service principal?

**Comment by chrey-gh** [comment]
Good point, John. This is a relict from the very beginning of this project. It's there, if you need key vault, you just would have to add the relevant accounts to these groups. But since we have UC (Unity Catalog) now, we don't need Key Vault. So I am inclined to get rid of the whole thing.

**Comment by chrey-gh** [comment]
in the new deployment there's no more keyvault
